packages/nextjs/app/blockexplorer/address/[address]/page.tsx

```diff
@@ -156,10 +42,9 @@ async function fetchByteCodeAndAssembly(buildInfoDirectory: string, contractPath
   return { bytecode, assembly };
 }
 
-export const getServerSideProps: GetServerSideProps = async context => {
-  const address = (context.params?.address as string).toLowerCase();
+export const getData = async (address: string) => {
   const contracts = deployedContracts as GenericContractsDeclaration | null;
-  const chainId = hardhat.id;
+  const chainId = 31337;
```

Comment:
return hardhat.id please

Reply:
I used the id directly , because we can't use dot operator in server Side components.
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/102049482/1f50f49c-b929-4e2f-a30c-739d8989ffa8)
What do you suggest we should do?

Reply:
I see, wagmi chains don't work great with server components.

Could you move `31337` to constant `HARDHAT_CHAIN_ID = 31337` outside this function and use that constant instead of `chainId`?

Reply:
done 👍